src/main/java/com/spotinst/sdkjava/model/api/ocean/kubernetes/ApiVirtualNodeGroupAutoScaleSpec.java

```diff
@@ -0,0 +1,40 @@
+package com.spotinst.sdkjava.model.api.ocean.kubernetes;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class ApiVirtualNodeGroupAutoScaleSpec {
```

Comment:
Rename to 'ApiK8sVngAutoScaleSpec' for consistency. Following same name convention throughout is better.